packages/manager/src/features/Kubernetes/CreateCluster/ClusterNetworkingPanel.test.tsx

```diff
@@ -0,0 +1,105 @@
+
+import { ClusterNetworkingPanel } from './ClusterNetworkingPanel';
+
+const queryMocks = vi.hoisted(() => ({
+  useRegionQuery: vi.fn().mockReturnValue({ data: { capabilities: ['VPCs'] } }),
+  useAllVPCsQuery: vi.fn().mockReturnValue({ data: [], isLoading: false }),
+  useIsLkeEnterpriseEnabled: vi.fn(() => ({
+    isLkeEnterprisePhase2FeatureEnabled: true,
+  })),
+}));
+
+vi.mock('@linode/queries', async () => {
+  const actual = await vi.importActual('@linode/queries');
+  return {
+    ...actual,
+    useRegionQuery: queryMocks.useRegionQuery,
+    useAllVPCsQuery: queryMocks.useAllVPCsQuery,
+  };
+});
+
+vi.mock('../kubeUtils', async () => {
+  const actual = await vi.importActual('../kubeUtils');
+  return {
+    ...actual,
+    useIsLkeEnterpriseEnabled: queryMocks.useIsLkeEnterpriseEnabled,
+  };
+});
+
+const props = {
+  selectedRegionId: 'us-east',
+};
+
+const defaultValues = {
+  stack_type: 'ipv4',
+  nodePools: [],
+};
+
+describe('ClusterNetworkingPanel', () => {
+  it('renders IP version options and VPC radio buttons when LKE-E phase2MTC feature is enabled', () => {
+    const { getByText } = renderWithThemeAndHookFormContext({
+      component: <ClusterNetworkingPanel {...props} />,
+      useFormOptions: {
+        defaultValues,
+      },
+    });
+
+    // Confirm stack type section
+    expect(getByText('IP Version')).toBeVisible();
+    expect(getByText('IPv4')).toBeVisible();
+    expect(getByText('IPv4 + IPv6')).toBeVisible();
+
+    // Confirm VPC section
+    expect(getByText('VPC')).toBeVisible();
+    expect(
+      getByText('Automatically generate a VPC for this cluster')
+    ).toBeVisible();
+    expect(getByText('Use an existing VPC')).toBeVisible();
+  });
+
+  it('selects correct default values for radio buttons', () => {
+    const { getByRole } = renderWithThemeAndHookFormContext({
+      component: <ClusterNetworkingPanel {...props} />,
+      useFormOptions: {
+        defaultValues,
+      },
+    });
+
+    // Confirm stack type default
+    expect(getByRole('radio', { name: 'IPv4' })).toBeChecked();
+    expect(getByRole('radio', { name: 'IPv4 + IPv6' })).not.toBeChecked();
+
+    // Confirm VPC default
+    expect(
+      getByRole('radio', {
+        name: 'Automatically generate a VPC for this cluster',
+      })
+    ).toBeChecked();
+
+    expect(
+      getByRole('radio', {
+        name: 'Use an existing VPC',
+      })
+    ).not.toBeChecked();
+  });
+
+  it('shows VPC and Subnet fields when "Use an existing VPC" is selected', async () => {
+    const { getByLabelText, findByLabelText } =
+      renderWithThemeAndHookFormContext({
+        component: <ClusterNetworkingPanel {...props} />,
+        useFormOptions: {
+          defaultValues,
+        },
+      });
+
+    await userEvent.click(getByLabelText('Use an existing VPC'));
+
+    // Confirm VPC options display
+    expect(await findByLabelText('VPC')).toBeVisible();
+    expect(await findByLabelText('Subnet')).toBeVisible();
```

Comment:
Do these need to be async? 

```suggestion
    // Confirm VPC options display
    expect(getByLabelText('VPC')).toBeVisible();
    expect(getByLabelText('Subnet')).toBeVisible();
```